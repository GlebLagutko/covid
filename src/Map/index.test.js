import {render} from '@testing-library/react';

import CovidMap from '.';
import React from "react";

describe('CovidMap', () => {

  it('renders correctly', async () => {

    const {container} = render(<CovidMap/>);

    expect(container).toMatchSnapshot();
  });
});