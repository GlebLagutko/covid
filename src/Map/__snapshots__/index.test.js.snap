// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CovidMap renders correctly 1`] = `<div />`;
